### ISSUE TITLE
Overlay issue causes boarderlands2 to crash on startup

### ISSUE THREAD

**Issue by Salvation-Son-Kun** [ValveSoftware/steam-runtime#231]
steam package version: 1496897923
built: jun 8 2017

Solus

Am not opted into steam beta

Have checked for uupdates

When trying to launch borderlands 2 it crashes on startup. The following is the only error that is output in terminal:

`ERROR: ld.so: object '/home/adam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.`

**Comment by smcv** [comment]
Indeed, looks like Borderlands bundles some libraries that it probably shouldn't: https://steamdb.info/depot/49533/

It isn't obvious to me which one specifically would be causing a crash.
